packages/cli/src/projects.ts

```diff
@@ -0,0 +1,28 @@
+import { Command } from 'commander';
+import { medplum } from '.';
+import { MedplumClient } from '@medplum/core';
+// import { prettyPrint } from './utils';
+
+export const project = new Command('project');
+
+project
+  .command('list')
+  .description('List of current projects')
+  .action(async () => {
+    projectList(medplum);
+  });
+
+function projectList(medplum: MedplumClient): void {
+  const logins = medplum.getLogins();
+
+  const projects = logins
+    .map((login) => {
+      login.project;
+      const keyValue = Object.entries(login.project);
+      const values = keyValue.map(([key, value]) => `${key}: ${value}`);
```

Comment:
This makes assumptions about the contents of `getLogins()` which might not always be true.

Rather than showing all values, let's be explicit about showing just the project details.

For example:

```ts
.map((login) => `${login.project.display} (${login.project.reference})`)
```